documentation/docs/developers/strategies/simple-trade.md

```diff
@@ -0,0 +1,12 @@
+# Simple Trade
+
+## Overview
+
+Simple trade strategy expands upon Perform Trade strategy. It has a few additional features such as time constraints and order specific loggings.
+
+#### Time restriction
+There is a time restriction `_time_delay` set that will ensure that there is a given amount of time in between the trades. When an order is placed in `c_process_market()`, it checks that the current timestamp is greater than previous order’s timestamp plus delay time. If it is a valid time to submit new order, the order will be processed. 
```

Comment:
Again, it's better to list out the differences in point form. It's easier for readers to scan through the documentation in that format.

You should also talk about how these two new attributes are related to the arguments in the constructor.